inst/odin/carehomes.R

```diff
@@ -1758,6 +1831,18 @@ kappa_diagnoses <- user() # ignore.unused
 phi_all_admission <- user() # ignore.unused
 kappa_all_admission <- user() # ignore.unused
 rho_pillar2_tests <- user() # ignore.unused
+kappa_pillar2_cases <- user() # ignore.unused
 phi_pillar2_cases <- user() # ignore.unused
 phi_pillar2_cases_weekend <- user() # ignore.unused
-kappa_pillar2_cases <- user() # ignore.unused
+phi_pillar2_cases_under15 <- user() # ignore.unused
+phi_pillar2_cases_weekend_under15 <- user() # ignore.unused
+phi_pillar2_cases_15_24 <- user() # ignore.unused
+phi_pillar2_cases_weekend_15_24 <- user() # ignore.unused
+phi_pillar2_cases_25_49 <- user() # ignore.unused
+phi_pillar2_cases_weekend_25_49 <- user() # ignore.unused
+phi_pillar2_cases_50_64 <- user() # ignore.unused
+phi_pillar2_cases_weekend_50_64 <- user() # ignore.unused
+phi_pillar2_cases_65_79 <- user() # ignore.unused
+phi_pillar2_cases_weekend_65_79 <- user() # ignore.unused
+phi_pillar2_cases_80_plus <- user() # ignore.unused
+phi_pillar2_cases_weekend_80_plus <- user() # ignore.unused
```

Comment:
Please make sure you end files with an empty line

Reply:
done